### DISCUSSION TITLE
Shared README for workspaces

### DISCUSSION THREAD

**Discussion by cristianrgreco** [npm/rfcs#713] [RRFC]
Hello!

I have a monorepo project which uses NPM workspaces.

At the root of the project I have a `README`. 
It is important to have the README in the root of the project for GitHub. The README contains links to external documentation.

I have a `packages` directory, which contains multiple publishable packages.
I want each package to have the same README as the one in the root of the project, but this isn't possible today.
I do not want to duplicate the root README to every package as it becomes a maintenance issue. Symlinks also do not work.

It would be great to be able to provide for example a relative link to the README to support this use case.

Thanks for your consideration!

**Comment by ljharb** [comment]
I use a prepack script to copy the readme and license etc from the root to subpackages, and npmignore them, and then it’s automatic.

**Reply by cristianrgreco** [reply]
Thanks for the response @ljharb. I build the projects using `npm build -ws`, I don't build anything for the root. In this case I'd need to add the same prepack script to every package, right? I'd like to avoid the repetition

**Reply by ljharb** [reply]
yes, that's right.

repetition is desirable in monorepos, imo, but you could always make each prepack script do `cd ../.. && npm run prepack` or something.

**Reply by cristianrgreco** [reply]
OK, thanks for your help!
